src/interfaces/todo-list-interface.ts

```diff
@@ -0,0 +1,4 @@
+export interface TodoListResourceIF {
+  title: string
+
```

Comment:
Does this blank line cause a lint error locally? `npm run lint`

Reply:
it does not show any error, still, it is good to fix. fixed it